analyzers/PapasSim.py

```diff
@@ -19,6 +19,7 @@
 from ROOT import TLorentzVector, TVector3
```

Comment:
If Papas.py is obsolete (since we have PapasSim.py), 
it should be removed. 
Make sure that the tutorial is working